docker/usr/local/package/intel-graphics-compiler.sh

```diff
@@ -33,6 +33,26 @@ echo "Configuring using cmake file ${CMAKELISTS_FILE}"
 # Clone the dependency
 git clone https://github.com/intel/vc-intrinsics vc-intrinsics
 
+# Download the patches
```

Comment:
Cant we just pipe the output of `curl` into `patch` instead of saving to file?

Reply:
We could. That seems reasonable. Thinking about it, there is a benefit to doing it like this though: the patches are named clearly, so we know which one is which, which might be convenient later. We could do that with comments though I suppose. Good point!

Reply:
Was just thinking for consistency with our other scripts (`install-from-source` and `espeak.sh`).